### PR TITLE
fix hard-coded placeholder text

### PR DESCRIPTION
**Pull request by samumist**
fix https://github.com/adaptlearning/adapt_authoring/issues/2309

### CODE REVIEW

File: frontend/src/modules/sidebar/templates/sidebarFilter.hbs [hunk 1]
```diff
@@ -13,7 +13,7 @@
         </div>
     </div>
     <div class="sidebar-filter-search">
-        <input type="text" placeholder="Filter Tags" class="sidebar-filter-search-input" />
+      <input type="text" placeholder="{{t 'app.filterbytags'}}" class="sidebar-filter-search-input" />
```
Comment (canstudios-nicolaw) [inline frontend/src/modules/sidebar/templates/sidebarFilter.hbs]: Thanks for the PR! Please could you fix the indentation on this line?

Comment (samumist) [reply frontend/src/modules/sidebar/templates/sidebarFilter.hbs]: @canstudios-nicolaw thanks for pointing this out. fixed.

**Comment by canstudios-nicolaw** [comment]
We need to change the destination of this branch to `release/bugpatch`

**Comment by samumist** [comment]
@canstudios-nicolaw Thanks. changed base.

**Pull request merged**